### DISCUSSION TITLE
配置消息推送--如果是飞书通知，需要配置哪个参数？

### DISCUSSION THREAD

**Discussion by zhangsh90** [RayWangQvQ/BiliBiliToolPro#755] [Q&A]
SendNotify.js的文件中有配置飞书机器人Key的参数，但没有定时推送消息到飞书，请问还需要配置哪里？

**Comment by Lucio-0408** [comment]
导入3个环境变量
[
	{
		"value": "{\"msg_type\":\"text\",\"content\":{\"text\":#content#}}",
		"name": "Ray_Serilog__WriteTo__8__Args__bodyJsonTemplate"
	},
	{
		"value": "#content#",
		"name": "Ray_Serilog__WriteTo__8__Args__placeholder"
	},
	{
		"value": "你的飞书机器人链接",
		"name": "Ray_Serilog__WriteTo__8__Args__api"
	}
]
